src/test/ui/error-codes/E0161.rs

```diff
@@ -29,7 +29,15 @@
 #![feature(box_syntax)]
 
 fn foo(x: Box<[i32]>) {
-    box *x; //~ ERROR E0161
+    box *x;
+    //[ast]~^ ERROR E0161
+    //[nll]~^^ ERROR E0161
+    //[zflags]~^^^ ERROR E0161
+    //[edition]~^^^^ ERROR E0161
+    //[astul]~^^^^^ ERROR E0161
+    //[nllul]~^^^^^^ ERROR E0161
+    //[zflagsul]~^^^^^^^ ERROR E0161
+    //[editionul]~^^^^^^^^ ERROR E0161
```

Comment:
heh. We really need to implement something where a `//~ ERROR` in a revisions test means "this applies to all revisons" ...

Reply:
@oli-obk 
Why, `^^^^^^`, `//~| ERROR` doesn't work with versions?

Reply:
nope

Reply:
yeah I wish it did

Reply:
I was just too lazy to fix it